private-templates-service/client/src/components/Dashboard/PreviewModal/TemplateInfo/VersionCard/VersionCard.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import { Template } from 'adaptive-templating-service-typescript-node';
+
+import {
+  CardManageButton,
+  CardTitle,
+  VersionCardHeader, 
+  VersionCardRowTitle,
+  DateWrapper,
+  VersionCardRow,
+  StatusWrapper,
+  VersionIcon, 
+  VersionWrapper
+} from './styled'
+
+import {
+  Card,
+  CardHeader,
+  CardBody, 
+  StatusIndicator, 
+  Status
+} from './../styled';
+
+
+interface Props {
+  template: Template;
+  templateVersion: string;
+}
+
+
+class VersionCard extends React.Component<Props, {}> {
```

Comment:
You can just remove the state if its empty, its not required

Reply:
Fixed!